src/Illuminate/Queue/QueueManager.php

```diff
@@ -233,6 +234,16 @@ public function setDefaultDriver($name)
     }
 
     /**
+     * Get the name of the queue prefix.
+     *
+     * @return string
+     */
+    public function getQueuePrefix()
+    {
+        return isset($this->app['config']['queue.prefix']) ? $this->app['config']['queue.prefix'] : null;
```

Comment:
Can't we do `return $this->app['config']['queue.prefix'] ?? null` here? IIRC Laravel now requires PHP >= 7, doesn't it? Or is it only on `master`?